lib/drivers/animasDriver.js

```diff
@@ -1644,28 +1644,31 @@ module.exports = function (config) {
 
       }
 
-      var suspend = null;
-      var resume = null;
       for(var s in data.suspendResumeRecords) {
         var suspendresumedatum = data.suspendResumeRecords[s];
 
-        suspend = cfg.builder.makeDeviceEventSuspend()
-          .with_deviceTime(suspendresumedatum.suspendDeviceTime)
-          .with_reason({suspended: 'manual'})
-          .set('index', suspendresumedatum.index);
-        cfg.tzoUtil.fillInUTCInfo(suspend, suspendresumedatum.suspendJsDate);
-        suspend = suspend.done();
-        postrecords.push(suspend);
-
         if(suspendresumedatum.resumeJsDate > suspendresumedatum.suspendJsDate) {
+          var suspendResume = cfg.builder.makeDeviceEventSuspendResume()
+            .with_deviceTime(suspendresumedatum.suspendDeviceTime)
+            .with_reason({suspended: 'manual', resumed: 'manual'})
+            .with_duration(suspendresumedatum.resumeJsDate - suspendresumedatum.suspendJsDate)
+            .with_payload({
+              resumeDeviceTime : suspendresumedatum.resumeDeviceTime
+            })
+            .set('index', suspendresumedatum.index);
+          cfg.tzoUtil.fillInUTCInfo(suspendResume, suspendresumedatum.suspendJsDate);
+          postrecords.push(suspendResume.done());
+        }
+        else {
```

Comment:
So remind me, when you set the clock back after a suspend, is the resume date the same Jan 1, 2007 as it is when there's no resume (e.g., before uploading) or is it just a date before the suspend? If the latter, do we want to treat it slightly differently? Stick the out-of-order resume in the payload at least?